platform/vs/docker-physyncd-vs.mk

```diff
@@ -0,0 +1,14 @@
+# docker image for vs physyncd
```

Comment:
i do not feel the physyncd docker should be under vs platform. This should be a generic docker and can be used by different asic platform. Then, vs platform just include it.

Reply:
Ok, suggest we make that move once we have a physical asic implementing gearbox?